mne/viz/tests/test_topomap.py

```diff
@@ -667,3 +668,26 @@ def test_plot_topomap_cnorm():
     msg = "vmax=2.5 is implicitly defined by cnorm, ignoring vmax=10.*"
     with pytest.warns(RuntimeWarning, match=msg):
         plot_topomap(v, info, vmax=10, cnorm=cnorm)
+
+
+def test_plot_bridged_electrodes():
+    """Test plotting of bridged electrodes."""
+    np.random.seed(42)
```

Comment:
no global seeds please

Reply:
I copied it from the test above so I went ahead and changed that too